raft/errors.go

```diff
@@ -28,10 +28,6 @@ var (
 	// ErrOutOfDateLog is returned when a candidate's log is not up to date.
 	ErrOutOfDateLog = errors.New("out of date log")
 
-	// ErrUncommittedIndex is returned when a stream is started from an
```

Comment:
No, the stream starts from the highest available index on the leader and the follower will trim its log or re-request a snapshot if needed.